Gridy/Controller/ImageEditorViewController.swift

```diff
@@ -86,9 +88,9 @@ class ImageEditorViewController: UIViewController, UIGestureRecognizerDelegate {
         panGestureRecognizer.delegate = self
         view.addGestureRecognizer(panGestureRecognizer)
         
-//        let rotationGestureRecognizer = UIRotationGestureRecognizer(target: self, action: #selector(rotateImageView(_:)))
-//        rotationGestureRecognizer.delegate = self
-//        view.addGestureRecognizer(rotationGestureRecognizer)
+        let rotationGestureRecognizer = UIRotationGestureRecognizer(target: self, action: #selector(rotateImageView(_:)))
+        rotationGestureRecognizer.delegate = self
+        view.addGestureRecognizer(rotationGestureRecognizer)
```

Comment:
Small details: this change should have been pushed in a different commit than "Third screen created"